tests/test_common.py

```diff
@@ -535,39 +513,24 @@ def test_get_disk_with_buffer_scale_down():
         )
     }
 
-    # Case 1: The max_size_gib is not specified, so we expect to use the entire disk
-    # Usage implies we require 4 TB * 2 nodes * 4x buffer == 32TB to meet buffer
+    # Case 1: Usage implies we require 4 TB * 2 nodes * 4x buffer == 32TB to meet buffer
     # Only 28TB is currently allocated, but we do *not* want to scale up to meet
     # desired buffer because of the `scale_down` intent
     needed_disk = RequirementFromCurrentCapacity(
         current_capacity=current_cluster_copy.zonal[0],
         buffers=buffers_copy,
-        instance_candidate=default_reference_shape,
     ).disk_gib
     expected_buffer = 4  # disk buffer
     assert needed_disk <= disk_utilization_gib.mid * cluster_size.mid * expected_buffer
     assert needed_disk == 14000 * cluster_size.mid
     assert needed_disk == 28000
 
-    # Case 2: Same as case (1) but the max_size_gib is specified.
```

Comment:
The max side gib test cases are no longer correct because they are handleded by min_count / node density in #171